src/components/Checkbox/Checkbox.module.scss

```diff
@@ -0,0 +1,78 @@
+@use '../../styles/boilerplate.scss'as *;
+
+.container {
+  @include prefix(user-select, none, webkit ms moz);
+  margin-bottom: 16px;
+  display: flex;
+  align-items: center;
+  position: relative;
+  cursor: pointer;
+
+  /* Hide the browser's default checkbox */
+  input {
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+    height: 0;
+    width: 0;
```

Comment:
Couldn't this just be `display: none`? The `cursor` property is probably redundant at least.

Reply:
display none would work visually, but screen reader wise and I think it would loose some of the hooks like on focus etc. So I tend to go with opacity to maintain all those things. 

this has some interesting commentary - https://webaim.org/techniques/css/invisiblecontent/

Reply:
Ah, I didn't consider screen readers. Thanks for the link.